cluster-configuration/services-configuration.yaml

```diff
@@ -101,6 +101,7 @@ prometheus:
   prometheus-port: 9091
   # port for node exporter
   node-exporter-port: 9100
+  scrape_interval: 30
```

Comment:
You should also modify `pai-management/quick-start/services-configuration.yaml.template`.

Otherwise quick start user will get jinja2 undefined variable exception.

Reply:
Solved